src/scenes/Gallery/components/Header.tsx

```diff
@@ -28,6 +29,16 @@ function Header({ usernameOrWalletAddress }: Props) {
           </Text>
         </StyledUserDetails>
       </StyledLeftContainer>
+      <StyledRightContainer>
+        <DropdownMenu
+          mainText="Edit Profile"
+          options={[
+            { label: 'Edit name', value: '/edit/name' },
+            { label: 'Edit bio', value: '/edit/bio' },
+            { label: '+ New Collection', value: '/add/collection' },
+          ]}
```

Comment:
again we would want to memoize this array, so that it's stable.

since this array is static and doesn't depend on any variables, we could move it outside of the component completely:

```js
const options = [{}, {}, {}]

function Header() {
    ...
    <DropdownMenu options={options}
}

Reply:
yup, good call. Do you think it makes sense to have the DropdownMenu links passed in with this format? ie just passing in two values, display text and the path?